### PR TITLE
Set the net to main by default

### PR DESCRIPTION
**Pull request by Liaojinghui**
Set the net to main by default in case the user does not set that param.

### CODE REVIEW

File: neo-cli/Settings.cs [hunk 1]
```diff
@@ -13,12 +14,30 @@ internal class Settings
...
-        static Settings()
+        static bool UpdateDefault(IConfiguration configuration)
         {
-            IConfigurationSection section = new ConfigurationBuilder().AddJsonFile("config.json").Build().GetSection("ApplicationConfiguration");
-            Default = new Settings(section);
+            var settings = new Settings(configuration.GetSection("ApplicationConfiguration"));
+            return null == Interlocked.CompareExchange(ref _default, settings, null);
+        }
+
+        public static bool Initialize(IConfiguration configuration)
```
Comment (shargon) [inline neo-cli/Settings.cs]: Why this are changed?

Comment (Liaojinghui) [reply neo-cli/Settings.cs]: This change is actually your previous modification, not my commit.  https://github.com/neo-project/neo-cli/pull/415/commits/626b1c8b319d735f7d880bda696306e4e52c1cd5

Comment (shargon) [reply neo-cli/Settings.cs]: But why appear?

Comment (Liaojinghui) [reply neo-cli/Settings.cs]: Guess this pull request is regarded as the child of my previous one, see, even the commit number is 11.

Comment (Liaojinghui) [reply neo-cli/Settings.cs]: Figured out, I didn't merge the latest update of this repo, fixed now.

**Comment by shargon** [comment]
Why we need `config.json` now?

**Comment by Liaojinghui** [comment]
> Why we need `config.json` now?

I don't think it is necessary.

> Why we need `config.json` now?

Just reviewed https://github.com/neo-project/neo/blob/master/neo/ProtocolSettings.cs feels like there is totally no need to add `--mainnet` command

**Comment by Liaojinghui** [comment]
Please ignore this pull request, the original code is already sound enough to handle all condition.

**Pull request closed without merging**